src/quacc/atoms/skzcam.py

```diff
@@ -1,73 +1,75 @@
 from __future__ import annotations
```

Comment:
You have to include this in order for type hints to be version-backwards compatible. We included it in all modules.